### PR TITLE
Fix Actions CI on MacOS

### PR DESCRIPTION
**Pull request by werebus**
(Probably)

### CODE REVIEW

**Comment by werebus** [comment]
"Style" failure is because this is the first time _our_ deploy config has been subject to _their_ Rubocop rules

File: Gemfile.lock [hunk 1]
```diff
@@ -180,6 +181,7 @@ GEM
     version_gem (1.1.4)
 
 PLATFORMS
+  arm64-darwin-22
```
Comment (bandogora) [inline Gemfile.lock:184]: It's weird that [the error](https://github.com/umts/strap/actions/runs/11977229119/job/33494615742#step:13:75) says `your local platform is arm64-darwin-22`, but the [Image Release](https://github.com/actions/runner-images/releases/tag/macos-14-arm64%2F20241119.509) says it's `Kernel Version: Darwin 23.6.0`.

https://github.com/umts/strap/actions/runs/11977229119/job/33494615742#step:1:12

Comment (bandogora) [reply Gemfile.lock:184]: Maybe add a `bundle platform` step and see if `arm64-darwin-23` is supported?

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -69,6 +69,7 @@ jobs:
         with:
           bundler-cache: true
 
+      - run: bundle platform
```
Comment (bandogora) [inline .github/workflows/tests.yml:72]: Don't forget to remove!

**Review by bandogora — APPROVED** [review]

**Pull request merged**